spec/models/user_spec.rb

```diff
@@ -0,0 +1,12 @@
+RSpec.describe User, type: :model do
+  describe 'associations' do
+    it { should have_many(:posts).with_foreign_key('author_id') }
+    it { should have_many(:comments).with_foreign_key('author_id') }
+    it { should have_many(:likes).with_foreign_key('author_id') }
+  end
+
+  describe 'validations' do
+    it { should validate_presence_of(:name) }
+    it { should validate_numericality_of(:posts_counter).only_integer.is_greater_than_or_equal_to(0) }
+  end
+end
```

Comment:
- [ ] Good job adding the tests @Becky449, But it's among the primary requirements to test every custom method in the models. Hence please add a test for the `three_latest_posts` method which is inside your `user` class.